bh/documentation/src/tab-navigation/examples/TabNavigationExample.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+    <bh-tabs v-model="activeTab">
+      <bh-tab-item label="Food">
+        Nunc nec velit nec libero vestibulum eleifend.
```

Comment:
Put a button with tooltip inside to test overflow.

Reply:
On tab-item called 'Music' there is a button with tooltip.